sixtyfps_compiler/fileaccess.rs

```diff
@@ -44,6 +48,10 @@ mod builtin_library {
 
     use super::VirtualFile;
 
+    pub(crate) fn styles() -> Vec<&'static str> {
```

Comment:
Why do we need this wrapper instead of calling builtin_styles() dirrectly?

Reply:
Changed this.

Reply:
And the indirection is back: widget_library() is not accessible outside the module it is included into.

Reply:
you can add the `pub(crate)` in the generated code.